.github/workflows/check.yaml

```
@@ -0,0 +1,34 @@
+name: check
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        emacs_version:
+          - 27.1
```

Good to include the latest stable release version

```suggestion
          - 28.2
          - 27.1
```

---

(Maybe I messed up tabs/spaces here, might want to check)